webplugin/pluginOptimizer.js

```diff
@@ -107,7 +107,11 @@ const compressAndOptimize = () => {
             path.resolve(__dirname, 'js/app/events/applozic-event-listener.js'),
             path.resolve(__dirname, 'js/app/events/applozic-event-handler.js'),
             path.resolve(__dirname, 'js/app/km-post-initialization.js'),
-            path.resolve(__dirname, 'js/app/mck-ringtone-service.js')
+            path.resolve(__dirname, 'js/app/mck-ringtone-service.js'),
```

Comment:
@Suraj-Kumar is it possible to add these files at run time only for the customers who enable this feature through javascript code.

Reply:
if we do that than it will append these file for every request which will increase API response time. this way it will append the file when server starts and response time remains constant. 
just a thought.